Update SimpliSafe documentation for event entity platform (#46353)
home-assistant/home-assistant.github.io: source/_integrations/simplisafe.markdown

| `light`                | yes      | Whether the light on the base station should display when armed              |
| `voice_prompt_volume`  | yes      | The volume of the base station's voice prompts                               |

## System events

Each SimpliSafe system provides a system event {% term entity %} that captures events from your security system with the following attributes:

- `event_type`: One of the following:
  - **Automatic test** (`automatic_test`)
  - **Device test** (`device_test`)
  - **Secret alert triggered** (`secret_alert_triggered`)
  - **Sensor paired and named** (`sensor_paired_and_named`)
  - **User initiated test** (`user_initiated_test`)
- `changed_by`: The PIN that triggered the event (if applicable)
- `info`: A human-friendly string describing the event in more detail
- `sensor_name`: The sensor that triggered the event (if applicable)
- `sensor_serial`: The serial number of the sensor that triggered the event (if applicable)

> - `sensor_type`: The type of sensor that triggered the event (if applicable)

Reviewer: Avoid consecutive colons and add periods:
```suggestion
- `event_type`, that should be one of the following:
  - **Automatic test** (`automatic_test`)
  - **Device test** (`device_test`)
  - **Secret alert triggered** (`secret_alert_triggered`)
  - **Sensor paired and named** (`sensor_paired_and_named`)
  - **User initiated test** (`user_initiated_test`).
- `changed_by`: The PIN that triggered the event (if applicable).
- `info`: A human-friendly string describing the event in more detail.
- `sensor_name`: The sensor that triggered the event (if applicable).
- `sensor_serial`: The serial number of the sensor that triggered the event (if applicable).
- `sensor_type`: The type of sensor that triggered the event (if applicable).
```

### Automation example: detecting secret alert events

For cases where the default {% term polling %} interval of 30 seconds is too long for automations, you can use secret alerts to get push notifications of a sensor being triggered.

> Home Assistant will automatically set the status to triggered for binary sensor devices that have secret alerts. However, due to the way SimpliSafe implements secret alerts, you can only receive push notifications when a device is triggered, not when they are cleared. Clearing a binary sensor can only be accomplished by polling.

Reviewer: If it is referring to a binary sensor device:
```suggestion
Home Assistant will automatically set the status to triggered for binary sensor devices that have secret alerts. However, due to the way SimpliSafe implements secret alerts, you can only receive push notifications when a device is triggered, not when it is cleared. Clearing a binary sensor can only be accomplished by polling.
```

## Events

> Each SimpliSafe system provides a system event {% term entity %} that captures events from your security system with the following attributes:

Author: Per the HA docs standards (https://developers.home-assistant.io/docs/documenting/standards), deprecated features get their docs removed, not a deprecation notice added. The old SIMPLISAFE_EVENT section was already removed.
